app/views/streams/main_stream.html.haml

```diff
@@ -44,20 +38,104 @@
         %li{data: {stream: "public"}}
           = link_to t("streams.public.title"), public_stream_path, rel: "backbone", class: "hoverable"
 
-    .col-md-6
-      #aspect_stream_container.stream_container
-        = render 'aspects/aspect_stream', stream: @stream
+      .info-bar.hidden-xs
+        #selected_aspect_contacts.section
+          .title
+            %h5.stream-title
+              = @stream.title
+          .content
+
+        - if AppConfig.settings.invitations.open?
+          .section
+            .title
+              %h5.title-header
+                = t('shared.invitations.invite_your_friends')
+            .content
+              = render "shared/invitations"
+
+        .section
+          .title
+            %h5.title-header
+              = t('aspects.index.new_here.title')
+          .content
+            != t('aspects.index.new_here.follow', link: link_to("#"+t('shared.publisher.new_user_prefill.newhere'), tag_path(name: t('shared.publisher.new_user_prefill.newhere'))))
+            %br
+            = link_to(t('aspects.index.new_here.learn_more'), "http://wiki.diasporafoundation.org/Welcoming_Committee")
+
+        .section
+          .title
+            %h5.title-header
+              = t('aspects.index.help.need_help')
+          .content
+            %p
+              = t('aspects.index.help.here_to_help')
+            %p
+              = t('aspects.index.help.do_you')
+            %ul
+              %li
+                != t('aspects.index.help.have_a_question', :link => link_to("#"+t('aspects.index.help.tag_question'), tag_path(:name => t('aspects.index.help.tag_question'))))
+              %li
+                != t('aspects.index.help.find_a_bug', :link => link_to("#"+t('aspects.index.help.tag_bug'), tag_path(:name => t('aspects.index.help.tag_bug'))))
+              %li
+                != t('aspects.index.help.feature_suggestion', :link => link_to("#"+t('aspects.index.help.tag_feature'), tag_path(:name => t('aspects.index.help.tag_feature'))))
+            %p
+              != t('aspects.index.help.tutorials_and_wiki',
+                   :faq => link_to(t('_help'), help_path),
+                   :tutorial => link_to(t('aspects.index.help.tutorial_link_text'), "https://diasporafoundation.org/tutorials", :target => '_blank'),
+                   :wiki => link_to('Wiki','http://wiki.diasporafoundation.org', :target => '_blank'), :target => '_blank')
+
+        - unless AppConfig.configured_services.blank? || all_services_connected?
+          .section
+            .title
+              %h5.title-header
+                = t('aspects.index.services.heading')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.